packages/mobile/src/pincode/PincodeLock.tsx

```diff
@@ -54,6 +64,22 @@ function PincodeLock() {
     }
   }, [])
 
+  const { error: getPinWithBiometryError } = useAsync(async () => {
+    if (shouldGetPinWithBiometry) {
+      const pin = await getPincodeWithBiometry()
+      void onCompletePin(pin)
+    }
+  }, [])
+
+  if (shouldGetPinWithBiometry && !getPinWithBiometryError) {
+    return (
+      <SafeAreaView style={styles.loadingContainer}>
+        <Image testID="BackgroundImage" source={background} style={styles.backgroundImage} />
+        <Logo type={LogoTypes.LIGHT} height={70} />
+      </SafeAreaView>
```

Comment:
This is trying to recreate the native app splash, right?
I think we should avoid doing that as it won't fully match it in term of position of the elements. Leading to a slight jump of things, which doesn't look very good.
Instead I propose we use maybe a blurview or wait for the biometry check to happen before dismissing the real splash (though this one probably is not so trivial, given how the app inits right now).

Reply:
yeah i wanted to recreate the native splash. i don't think the jump is noticeable - actually we can't even see the logo because the native animation covers the central part of the screen. i could remove the logo? i just need the background really, to give the illusion that the biometric verification is happening while the app is loading

Reply:
Yes let's remove the logo here as a compromise then.
Maybe one day we'll have a more direct app init where we could do the biometry check before we trigger the call to dismiss the native splash.

Reply:
sounds like a good compromise, i have removed the logo :)